circle.yml

```diff
@@ -13,7 +13,10 @@ dependencies:
     - mix compile
 test:
   override:
-    - mix test --include requires_env --cover
+    - |
+      if [ ${CIRCLE_PR_USERNAME} ]; then MIX_ENV=test mix test --cover; else MIX_ENV=test mix test --include requires_env --cover; fi
```

Comment:
Out of curiosity, is there a benefit to having `CIRCLE_PR_USERNAME` set for some but not others? Would it be reasonable for the test suite to run the same way for everyone?